confgenerator/testdata/valid/linux/logging-receiver_tomcat/input.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2021 Google LLC
```

Comment:
Can we split this unit test to two? One for the simplest config that works with out of the box Tomcat, and one with the more advanced one. Some users use the valid test cases as sample config reference.

Reply:
Added.

Reply:
Looks great. Thanks!